### ISSUE TITLE
Confirmations checkboxes and detailed information

### ISSUE THREAD

**Issue by voroyam** [owncloud/encryption#26]
With encryption being a main source of (self inflicted = ?) problems users are encountering right now I suggest some changes.

First that is the current process that users take to enable encryption in the web UI:

![en1](https://user-images.githubusercontent.com/24212810/35390341-582ea044-01db-11e8-9336-2eeb260f4461.png)
![en2](https://user-images.githubusercontent.com/24212810/35390282-21eca85a-01db-11e8-9826-0c32859cee16.png)
![en3](https://user-images.githubusercontent.com/24212810/35390283-22069fa8-01db-11e8-89bf-02c3102fac57.png)
![en4](https://user-images.githubusercontent.com/24212810/35390284-22218bec-01db-11e8-8eec-aa0ccc50d046.png)
![en5](https://user-images.githubusercontent.com/24212810/35390285-2252435e-01db-11e8-8bd1-8b8eb6bd32ac.png)

What I am suggesting:

When enabling encryption - implement a **check box**, that **has to be checked** in order to press the button to enable encryption. 
The check box could say "I have read and understood the consequences and want to proceed". 
That way the user can't say - "I did not read or understood what I have done", because he had checked the check box :-D

When choosing mater key or user key as the modules - there are **no warnings what so ever**. Thats not good.

The documentation ( which very few read ) tells you in one short sentence without a warning label 
"It is not possible to disable it."

https://doc.owncloud.com/server/10.0/admin_manual/configuration/files/encryption_configuration.html#create-a-new-master-key

I suggest implementing a warning and a check box when choosing the modules.

Master key encryption has optimally has to be set up on a fresh installation. That is mentioned in the docs but not on the web UI.

> A fresh installation with no existing data

What to do if you already have data on your server is also missing in the web UI.

Basically what I want is some warning that jumps at you when you are choosing the encryption module.

> You are about to do this:
> 
> This is a serious action which can have **dramatic** consequences.
> 
> Read this https://doc.owncloud.com/server/10.0/admin_manual/configuration/files/encryption_configuration.html#create-a-new-master-key
> 
> You **can't go back**
> Your data may be **lost** if you lose your password
> Are you really **sure** you want to do this? 
> 

Something along those lines.
